building/devices/zynqmp.rst

```diff
@@ -1,150 +1,74 @@
 .. _zynqmp:
 
 #########################
-ZynqMP zcu10x and Ultra96
+Zynq MPSoC
 #########################
```

Comment:
I think you need to update  `#####... ` to the section title size (I'm not sure but it's the case on other .rst files).

```
##########
Zynq MPSoC
##########
(...)

Supported boards
****************
(...)